doc/man3/ASYNC_WAIT_CTX_new.pod

```diff
@@ -4,13 +4,16 @@
 
 ASYNC_WAIT_CTX_new, ASYNC_WAIT_CTX_free, ASYNC_WAIT_CTX_set_wait_fd,
 ASYNC_WAIT_CTX_get_fd, ASYNC_WAIT_CTX_get_all_fds,
-ASYNC_WAIT_CTX_get_changed_fds, ASYNC_WAIT_CTX_clear_fd - functions to manage
-waiting for asynchronous jobs to complete
+ASYNC_WAIT_CTX_get_changed_fds, ASYNC_WAIT_CTX_clear_fd,
+ASYNC_WAIT_CTX_set_callback, ASYNC_WAIT_CTX_get_callback,
+ASYNC_WAIT_CTX_set_status, ASYNC_WAIT_CTX_get_status, ASYNC_callback_fn
+- functions to manage waiting for asynchronous jobs to complete
```

Comment:
There is some discussion about ASYNC_WAIT_CTX and file descriptors on the ASYNC_start_job page which probably also needs to be updated.

Reply:
Done. Add in ASYNC_start_job as well.